Cargo.toml

```
@@ -9,13 +9,16 @@ readme = "README.md"
 license = "MIT OR Apache-2.0"
 
 [dependencies]
+cached = "0.20.0"
```

This pulls in `futures` by default, which we don't need.
```suggestion
cached = { version = "0.20.0", default-features = false }
```
(maybe I should make a PR disabling default features in cached?)